File: app/src/main/java/com/example/musicwiki/repo/network/albumsinfo/Tag.java

package com.example.musicwiki.repo.network.albumsinfo;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.Objects;

public class Tag {
    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (!(o instanceof Tag)) return false;
        Tag tag = (Tag) o;
        return Objects.equals(getUrl(), tag.getUrl()) && Objects.equals(getName(), tag.getName());
    }

    @Override
    public int hashCode() {
        return Objects.hash(getUrl(), getName());
    }

    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("name")
    @Expose
    private String name;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/com/example/musicwiki/repo/network/genre/Tag.kt
package com.example.musicwiki.repo.network.genre

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

data class Tag(
    @SerializedName("name")
    @Expose
    var name: String? = null,

    @SerializedName("count")
    @Expose
    var count: Int? = null,

    @SerializedName("reach")
    @Expose
    var reach: Int? = null
)